tests/ide/recordflux.gpr

```diff
@@ -0,0 +1,7 @@
+project RecordFlux is
+
+   for Create_Missing_Dirs use "True";
+   for Source_Dirs use (".", "../../specs");
+   for Languages use ("Ada", "RecordFlux");
+
+end RecordFlux;
```

Comment:
It would be nice if Ada and RecordFlux files could be used in the same project.

```diff
diff --git a/test.gpr b/test.gpr
index e3ba691..4a4e71c 100644
--- a/test.gpr
+++ b/test.gpr
@@ -6,12 +6,14 @@ project Test is
    AUnit : Flag_Type := external ("aunit", "yes");
    Optimization : Flag_Type := external ("optimization", "no");
 
+   for Languages use ("Ada", "RecordFlux");
+
    case AUnit is
       when "yes" =>
-         for Source_Dirs use ("generated", "tests");
+         for Source_Dirs use ("generated", "tests", "specs");
          for Main use ("test.adb");
       when "no" =>
-         for Source_Dirs use ("generated");
+         for Source_Dirs use ("generated", "specs");
    end case;
 
    for Object_Dir use "build";
```

But currently the `Build All` action will fail then:

```Console
gprbuild -d -P/[...]/RecordFlux/test.gpr -j0
test.gpr:4:09: warning: there are no sources of language "RecordFlux" in this project
```

Reply:
You can do that by telling `gprbuild` that RecordFlux is a language:

```Diff
--- a/test.gpr
+++ b/test.gpr
@@ -6,12 +6,18 @@ project Test is
    AUnit : Flag_Type := external ("aunit", "yes");
    Optimization : Flag_Type := external ("optimization", "no");
 
+   for Languages use ("Ada", "RecordFlux");
+
+   package Compiler is
+      for Driver ("RecordFlux") use "";
+   end Compiler;
+
    case AUnit is
       when "yes" =>
-         for Source_Dirs use ("generated", "tests");
+         for Source_Dirs use ("generated", "tests", "specs");
          for Main use ("test.adb");
       when "no" =>
-         for Source_Dirs use ("generated");
+         for Source_Dirs use ("generated", "specs");
    end case;
 
    for Object_Dir use "build";
```

Reply:
> You can do that by telling gprbuild that RecordFlux is a language

It would be nice if that changes could be added to the `test.gpr`.

Reply:
Done in 9e9f48e.